cypress/support/config/services.js

```diff
@@ -39,8 +39,12 @@ const genServices = {
       },
       mediaAssetPage: {
         paths: {
-          live: ['/afaanoromoo/oduu-51248626'],
-          test: ['/afaanoromoo/23149891'],
+          live: [
```

Comment:
beginning to think something like this would be a good idea 🤔one for the future possibly!

```js
      mediaAssetPage: {
        envs: {
          live: {
            paths: ['/afaanoromoo/oduu-51248626'],
            enabled: true,
          },
          test: {
            paths: ['/afaanoromoo/23149891'],
            enabled: false,
          },
          local: {
            paths: ['/afaanoromoo/23149891'],
            enabled: true,
          },
        },
        smoke: false,
      },
```

Reply:
I like that idea - definitely one for a future PR

Reply:
I have implemented this now